mitodl/open-discussions
# do not spam check moderators

#### Pre-Flight checklist


#### What are the relevant tickets?
closes https://github.com/mitodl/open-discussions/issues/3071

#### What's this PR do?
This pr only runs the `check_post_for_spam` and `check_comment_for_spam` jobs if the comment/post author is not a moderator

#### How should this be manually tested?
Verify that you can create and edit posts as comments both when you are logged in as a moderator and when you are logged in as a normal user. Verify that the  `check_post_for_spam` and `check_comment_for_spam` jobs only run for regular users, not moderatiors.
